### PR TITLE
Add vs_lsmash chunk method for mkv/mp4/m2ts 

### PR DESCRIPTION
**Pull request by wwww-wwww**
now it can be the recommended method

### CODE REVIEW

**Review by master-of-zen — APPROVED** [review]

**Comment by n9Mtq4** [comment]
@wwww-wwww Did you test this on Windows or a Unix based OS?

File: Chunks/chunk_queue.py [hunk 1]
```diff
@@ -131,8 +132,19 @@ def create_video_queue_hybrid(args: Args, split_locations: List[int]) -> List[Ch
...
+    return create_video_queue_vs(args, split_locations, script)
+
+
+def create_video_queue_vslsmash(args: Args, split_locations: List[int]) -> List[Chunk]:
+    script = "from vapoursynth import core\n" \
+             "core.lsmas.LWLibavSource(\"{}\", cachefile=\"{}\").set_output()"
+    return create_video_queue_vs(args, split_locations, script)
+
+
+def create_video_queue_vs(args: Args, split_locations: List[int], script: List[str]) -> List[Chunk]:
```
Comment (n9Mtq4) [inline Chunks/chunk_queue.py:147]:
```suggestion
def create_video_queue_vs(args: Args, split_locations: List[int], script: str) -> List[Chunk]:
```

Can you also add the script param to the docstring for this function?

**Review by n9Mtq4 — CHANGES_REQUESTED** [review]

File: Startup/setup.py [hunk 2]
```diff
@@ -49,9 +49,25 @@ def check_exes(args: Args):
...
-        terminate()
+    if args.chunk_method in ['vs_ffms2', 'vs_lsmash']:
+        if not find_executable('vspipe'):
+            print('vspipe executable not found')
+            terminate()
+
+        try:
+            import vapoursynth
+            plugins = vapoursynth.get_core().get_plugins()
+        except:
```
Comment (n9Mtq4) [inline Startup/setup.py:60]:
```suggestion
        except ModuleNotFoundError:
```
This is to address [PEP 8: E722](https://www.flake8rules.com/rules/E722.html).

**Review by n9Mtq4 — CHANGES_REQUESTED** [review]

**Comment by luigi311** [comment]
Perhaps we should start adding tests to the workflow when new features are added. I need to add more tests for other encoders so if @wwww-wwww doesnt want to add it on this PR i will add it when i get around to the other encoders.

**Comment by master-of-zen** [comment]
@wwww-wwww :1st_place_medal:

**Pull request merged**